src/Keboola/DbExtractor/DbRetryProxy.php

```diff
@@ -18,7 +18,7 @@ class DbRetryProxy extends RetryProxy
     /* set initial backoff to 1 second */
     public const DEFAULT_BACKOFF_INTERVAL = 1000;
 
-    public const DEFAULT_EXCEPTED_EXCEPTIONS = ['PDOException', 'ErrorException'];
+    public const DEFAULT_EXPECTED_EXCEPTIONS = ['PDOException', 'ErrorException'];
```

Comment:
Haha, I thought that's weird, but thought that it's like "throw on everything EXCEPT those" :D

Reply:
Anyway DEFAULT_RETRIABLE_EXCEPTIONS ;) But to need to change it.